cmd/antrea-agent/agent.go

```diff
@@ -132,6 +132,11 @@ func run(o *Options) error {
 	egressEnabled := features.DefaultFeatureGate.Enabled(features.Egress)
 	enableAntreaIPAM := features.DefaultFeatureGate.Enabled(features.AntreaIPAM)
 	enableBridgingMode := enableAntreaIPAM && o.config.EnableBridgingMode
+	enableNodePortLocal := features.DefaultFeatureGate.Enabled(features.NodePortLocal) && o.config.NodePortLocal.Enable
+	enableMulticluster := features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.Enable
+	if !enableMulticluster && o.config.Multicluster.EnableStretchedNetworkPolicy {
```

Comment:
We should return an error, when o.config.Multicluster.Enable is true but features.Multicluster is not enabled.

And if o.config.Multicluster.Enable is false, but o.config.Multicluster.EnableStretchedNetworkPolicy is true, we may just reset it to false (I am fine to return an error too. Up to you).

Reply:
Changed to return error in validate func in option.go